beacon-chain/sync/subscriber_data_column_sidecar.go

```diff
@@ -52,3 +58,59 @@ func (s *Service) receiveDataColumnSidecar(ctx context.Context, sidecar blocks.V
 
 	return nil
 }
+
+// triggerGetBlobsV2ForDataColumnSidecar triggers getBlobsV2 retry when receiving a data column sidecar.
+// This function attempts to fetch the block and trigger the execution service's retry mechanism.
+func (s *Service) triggerGetBlobsV2ForDataColumnSidecar(ctx context.Context, blockRoot [32]byte) error {
```

Comment:
What's the purpose of this function if it is unable to reconstruction sidecars from EL if the block is not yet received?
The idea is to get blobs from the EL, reconstruct needed sidecars and reseed them even if we don't have (yet) the corresponding block.